src/src/pages/OtakuAnimes/main.js

```diff
@@ -0,0 +1,78 @@
+export const OtakuAnimes = {
+    name: 'OtakuAnimes',
+    domain: 'https://otakuanimess.com',
```

Comment:
Remove all the extra js and map files

Reply:
I need remove the whole folder `OtakuAnimes` inside src/src/pages, you mean?

Reply:
No you manully compiled the TS files or something and now  there are .js .map files everywhere

Reply:
Ah, ok ok